src/Maestro/Client/src/Generated/Models/SubscriptionUpdate.cs

```diff
@@ -8,6 +8,7 @@
 namespace Microsoft.DotNet.Maestro.Client.Models
```

Comment:
This file is generated off of the swagger API, it isn't manually edited.  So you add the right controller to SubscriptionsController, then do
```
npm install; npm start
```
From `src\Maestro\Client\generator`